src/Reader/Feed/FeedInterface.php

```diff
@@ -51,6 +51,13 @@ public function getDateCreated();
      */
     public function getDateModified();
 
+    /**
+     * Get the feed lastBuild date
+     *
+     * @return null|DateTime
+     */
+    public function getLastBuildDate();
```

Comment:
Please only focus on adding psalm instead of adding features like this.
We should not add new methods to interfaces. If there are issues with psalm, just add these to the baseline.

Would love to see this as a separated PR. *Maybe* we will add it to the next release tho.